content/items.yaml

```diff
@@ -0,0 +1,23 @@
+items:
+- question: What do these maps show?
+  answer: This atlas shows several different model projections of the European climate in 2050, illustrating the effect of constraining the projections. Not all climate simulations are equally likely, however this is not routinely addressed in current climate projections. Methods of isolating the more likely projections include comparing a simulation of the past to observed climate data recorded at the time. This can suggest how well the simulation may project the future. This process is known as ‘constraint’ and can affect the results given by an ensemble of climate simulations by ruling out the poorer simulations.
```

Comment:
There was a discussion in the email thread about potentially adding a figure to explain how "constraining methods" work. Is this still something we want to add?